config/app/com/microsoft/azure/iotsolutions/uiconfig/webservice/v1/controllers/PackageController.java

```diff
@@ -85,8 +109,25 @@ public PackageController(IStorage storage) {
 
         final String content = new String(Files.readAllBytes(file.getFile().toPath()));
         final String packageType = data.get(PACKAGE_TYPE_PARAM)[0];
+        String configType = data.get(PACKAGE_CONFIG_TYPE_PARAM)[0];
+
+        if (packageType.equals(PackageType.edgeManifest.toString()) &&
+                !(StringUtils.isBlank(configType)))
```

Comment:
should this also check if config type is null?

Reply:
isBlank does both.